tests/src/UnoTest/General/ValueTypes.uno

```diff
@@ -0,0 +1,38 @@
+using Uno;
+using Uno.Testing;
+
+namespace UnoTest.General
+{
+    struct Value
+    {
+        int A, B;
+
+        public Value(int a, int b)
+        {
+            A = a;
+            B = b;
+        }
+
+        public override int GetHashCode()
+        {
+            return base.GetHashCode();
+        }
+
+        public override bool Equals(object o)
+        {
+            return base.Equals(o);
+        }
+    }
+
+    class ValueTypes
+    {
+        [Test]
+        public static void Test()
```

Comment:
Could you mark this as `[Ignore]` until the test will pass? That way, we don't get broken bisections past this point...

Reply:
Sure, hold on!

Reply:
Done.